api.swagger.yaml

```
@@ -6560,6 +6715,420 @@ components:
+          type: array
+          items:
+            oneOf:
+              - $ref: '#/components/schemas/list-list-content-schema'
+              - $ref: '#/components/schemas/list-list-site_section-schema'
+              - $ref: '#/components/schemas/list-list-link-schema'
+            discriminator:
+              propertyName: type
+              mapping:
+                content: '#/components/schemas/list-list-content-schema'
+                site_section: '#/components/schemas/list-list-site_section-schema'
+                link: '#/components/schemas/list-list-link-schema'
+    list-list-common-schema:
+      type: object
+      properties:
+        type:
+          description: 'The type of content in the list.'
+          type: string
+          enum:
+            - 'content'
+            - 'site_section'
+            - 'link'
+        title:
+          description: 'Title of the content or link.'
+          type: string
+          example: 'More information here.'
+      required:
+      - type
+      - content
+    list-list-content-schema:
+      allOf:
+      - $ref: '#/components/schemas/list-list-common-schema'
+      - type: object
+        properties:
+          type:
+            example: 'content'
+          content:
+            type: object
+            properties:
+              uuid:
+                description: 'Invotra UUID of the content.'
+                type: string
+                format: uuid
+                example: '01234567-89ab-cdef-1234-56789abcdef0'
+    list-list-site_section-schema:
+      allOf:
+      - $ref: '#/components/schemas/list-list-common-schema'
+      - type: object
+        properties:
+          type:
+            example: 'site_section'
+          content:
+            type: object
+            properties:
+              uuid:
+                description: 'Invotra UUID of the site section.'
+                type: string
+                format: uuid
+                example: '01234567-89ab-cdef-1234-56789abcdef0'
+    list-list-link-schema:
+      allOf:
+      - $ref: '#/components/schemas/list-list-common-schema'
+      - type: object
+        properties:
+          type:
+            example: 'link'
+          content:
+            type: string
+            description: 'The URL of the link.'
+            example: 'https://example.com'
+    user-trimmed-schema:
+      properties:
+        uuid:
+          description: 'Invotra UUID of the user.'
+          type: string
+          format: uuid
+          example: '01234567-89ab-cdef-1234-56789abcdef0'
+        username:
+          description: The user's username
+          type: string
+          example: Dorothy Whitehead
+        title:
+          default: ''
+          description: The user's title
+          enum:
+            - ''
+            - Dr
+            - Mr
+            - Miss
+            - Mrs
+            - Ms
+          type: string
+          example: Mr
```

```suggestion
          example: Ms
```
for consistency with the adjacent example firstname